### ISSUE TITLE
Should we upload to the Youtube and send the link to our data holder?

### ISSUE THREAD

**Issue by YIHEEdinburgh** [ElvisJiang24/DS4D-GroupProject#32]
I think we should send the video to our data holder and help her to introduce the modernist library, should we?

**Comment by ElvisJiang24** [comment]
Yeah we can. Good practice to gather more useful feedback to help us update our creation, also a good way to attract more readers to use LibraryThings.com
